pipeline/console/container/push.py

```diff
@@ -86,7 +86,11 @@ def push_container(namespace: Namespace):
 
     docker_client: docker.DockerClient = docker.from_env(timeout=600)
 
-    registry_info = http.get(endpoint="/v4/registry")
+    registry_params = {}
+    if pipeline_config.extras and "turbo_registry" in pipeline_config.extras:
```

Comment:
think we should implement the same check as above, i.e.
```suggestion
    if pipeline_config.extras and pipeline_config.extras.get("turbo_registry", False):
```

Reply:
whoops, missed this one